### DISCUSSION TITLE
Phase 8 - The Plan

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#671] [Announcements - Phases]
I would like to be careful with Phase 8 and start with just two tags:

~James's location enhancements~ - #668 
NathanG's follow tag - #609
~Image tag~  - #694

**Comment by samsethi** [comment]
Hi Dave 

We have implemented the new location tag as originally proposed by Daniel J Lewis and refined by James Cridland. Below are four recent episodes from Podnews Daily showing James' different recording locations.  e.g https://truefans.fm/podnews-daily/671f5681d55de476b7cb4d4a

If you click the location tag we show it on a map but our next feature is to add multiple podcasts on a map so you can see podcasts nearby and filter by category. 

<img width="614" alt="Cursor_and_TrueFans___Shows___Podcast_Days_takes_place_in_Madrid_🔊" src="https://github.com/user-attachments/assets/4a477d70-0433-4564-af93-a47bc23e88da">

<img width="681" alt="TrueFans___Shows____It’s_a_Continent__is_Independent_Podcast_of_the_year_🔊" src="https://github.com/user-attachments/assets/eb3d24ad-d45f-4f11-8d44-f6f0ede1828b">

<img width="735" alt="Cursor_and_TrueFans___Shows___The_world’s_largest_podcast_agency_is_created_🔊" src="https://github.com/user-attachments/assets/fd314bc9-9503-4c03-a86f-d48391c93cf5">

<img width="646" alt="TrueFans___Shows___Women_in_Podcasting_Awards_winners_🔊" src="https://github.com/user-attachments/assets/c8fc1647-9452-4c3e-b935-c503e3669ebd">

**Comment by samsethi** [comment]
Hi Dave

1. [**Podcast Person Multiple Roles**](https://github.com/Podcastindex-org/podcast-namespace/discussions/498 (enhancement)). 
In TrueFans we have already enabled this feature. So a person can be a singer and songwriter or a person can be host and producer. 

<img width="988" alt="TrueFans___Shows___People_Pleaser" src="https://github.com/user-attachments/assets/7de067a9-7c3a-4891-a7c7-69a64dc3744c">

It is also essential we extend the person taxonomy as it is heavily focused on the podcast roles and not music or other roles. We have also extended the Person Taxomony in TrueFans to include songwriter, singer, AI, DJ (for musicasts), VJ (ValueJocks or VideoJocks) etc.

We have also added Person=AI so we can tag podcasts as AI.  We have added an AI filter to the user settings to hide AI hosted podcasts/episodes similar to the way we filter out podcasts tagged explicit but only if the user chooses.  

<img width="449" alt="Fullscreen_02_11_2024__21_14" src="https://github.com/user-attachments/assets/dcabb78a-a594-40a2-9d88-563115f300ff">

2. [**Podcast Pinning**](https://github.com/Podcastindex-org/podcast-namespace/discussions/570) this tag seems to fall between the cracks every phase. TrueFans enabled this at the same time we enabled Podrolls. In our admin dashboard, we enabled podcast creators to have their trailer episode as the default pinned episode or to choose another episode to feature. Adding this to the RSS feed and host to build a similar UI would be simple. So if the podcaster does nothing then the trailer is the featured/pinned podcast episode but if the podcaster picks another episode then that is pinned to the top.  So this could be a tag called pinned with a Remote Item to the featured/Pinned episode. 

![image](https://github.com/user-attachments/assets/d96b9c4c-6322-4a1b-ac8b-93ed3b78e7d4)


I hope this helps @daveajones grow Phase 8.

**Reply by picard102** [reply]
Seconding extending the Person tag. This has been a long time coming but keeps getting skipped.

**Comment by samsethi** [comment]
Hi Dave 

https://github.com/Podcastindex-org/podcast-namespace/discussions/570 seems to fall between the cracks
TrueFans enabled this at the same time we enabled Podrolls. In our admin dashboard we enabled podcast creators to have their trailer episode as the default pinned episode or to choose another episode to feature. Adding this to the RSS feed and host to build a similar UI would be simple.

Admin for choosing a featured episode.  By default we use the trailer as the featured episode.  
<img width="1328" alt="TrueFans___Dashboard" src="https://github.com/user-attachments/assets/68760bc8-461b-4c17-bdc6-f43e04c6050f" />

Exmaple of Featured Epiosde 
<img width="1216" alt="TrueFans___Shows___This_is_Gavin_Newsom" src="https://github.com/user-attachments/assets/6db0fcbf-54d7-4f64-9f12-6c77ad0a43af" />

**Comment by picard102** [comment]
What's the status of Phase 8? We're coming up on 10 months.

**Reply by picard102** [reply]
We're over year now. Is this project dead?

**Reply by MrMamen** [reply]
Sadly, it seems to be!

**Reply by Marzal** [reply]
The project is not dead. The creation of new tags is stopped for a while.
Which is good in order to Apps and Hosting to catch up and implement what is already specified.

**Reply by picard102** [reply]
Where was that communicated?

**Reply by Marzal** [reply]
Between Podcasting 2.0 podcast, Podcastindex.social, PI Github discussions ... and facts.

There is still discussions about improving tags, creating a new one in order to fix the boost metadata. A lot of work in the infrastructure is being made
Just because there is not new X tags each year, the project is not dead. Far from it.

You can say it moves slow, as this is a community project and nobody works full-time on it. And yes you need to follow both the Mastodon conversations and the podcast to be up-to-date, not ideal I know, but again. No big corporation with PR with a salary working here.

<img width="1209" height="1101" alt="imagen" src="https://github.com/user-attachments/assets/47381755-9015-4835-9fb8-841c12fc99ea" />

**Reply by picard102** [reply]
>Just because there is not new X tags each year, the project is not dead. Far from it.

I would say it's not the lack of new tags that lead people to think it's dead, but the significant periods of inactivity on the main discussion space for the project.

**Reply by Marzal** [reply]
Yes on Github there is no stability about the discussions. Dave said a while ago that didn't like the GH system and that he prefers discuss on Mastodon. For people who are not following enough P20 accounts it's easy to miss some conversations.

I did proposed a while back to make a kind of bridge and keep the GH more up-to-date and in order. [But that didn't take off](https://github.com/Podcastindex-org/podcast-namespace/discussions/695).
